### ISSUE TITLE
Session file generating tool

### ISSUE THREAD

**Issue by gonuke** [svalinn/DAGMC-viz#2]
Inputs: H5M geometry, mesthal (or equiv from other code), session template
Output: STL geometry without graveyard, VTK data, session file

python or bash script??? (basically a text parsing/replace)
* convert H5M to STL
* convert meshtal to VTK
* search/replace data filenames from session file template

option to only write session file or launch visit with that file
Automatic windows, linked for navigation: Pseudo color of tally, contour of error (Default contours), XY slice, YZ slice, ZX slice, Clip quadrant 1 at center

Option: Tally tagname, Error tagname

**Comment by gonuke** [comment]
Not in scope for this issue: automatic resetting of view depending on data extents

**Comment by gonuke** [comment]
After #4 consider tuning the session file to represent the STL in an optimal way. e.g. mesh edges opaque for curves and for surfaces: edges transparent, facets translucent
